Backend/Core/Infrastructure.Http/Users/IdentityUserRepository.cs

```diff
@@ -25,8 +25,8 @@ public IdentityUserRepository(IIdentityUsersAPI identityUsersAPI)
         public async Task Add(IdentityUser entity)
         {
             var userModel = Mapper.Map<UserModel>(entity);
-            var createUserModel = await IdentityUsersAPI.Create(userModel);
-            entity.Id = createUserModel.Id;
+            var createdUserModel = await IdentityUsersAPI.Create(userModel);
+            entity.Id = createdUserModel.Id;
```

Comment:
I don't know why, but I'm worried about this assigning. It's not an evident assigning for this method.